codec-http/src/main/java/io/netty/handler/codec/http/HttpConstants.java

```diff
@@ -39,7 +39,7 @@
     /**
      * Equals '='
      */
-    public static final byte EQUALS = 61;
+    public static final char EQUALS = '=';
```

Comment:
thats a breaking change so it cant be done in 4.1

Reply:
Yeah, thought about that after submission. 5.0?